### ISSUE TITLE
FixedWidthWriter does not honour FixedWidthWriterSettings.setNullValue

### ISSUE THREAD

**Issue by aldettinger** [uniVocity/univocity-parsers#238]
Hi guys,

  Does the failing test below demonstrate an issue ? Or is it the expected behavior ?
```java
import java.io.StringWriter;

import org.junit.Assert;
import org.junit.Test;

import com.univocity.parsers.fixed.FixedWidthFields;
import com.univocity.parsers.fixed.FixedWidthWriter;
import com.univocity.parsers.fixed.FixedWidthWriterSettings;

public class FixedWidthWriterNullValueTest {

    @Test
    public void writeNullRowShouldReplaceWithNullValueFromSettings() {
        FixedWidthWriterSettings settings = new FixedWidthWriterSettings(new FixedWidthFields(5));
        settings.setNullValue("N/A");
        StringWriter sw = new StringWriter();
        FixedWidthWriter fixedWidthWriter = new FixedWidthWriter(sw, settings);
        fixedWidthWriter.writeRow(new String[] {null});
        Assert.assertEquals("N/A  \n", sw.toString());
    }
}
```

Alex

**Comment by jbax** [comment]
Thanks for reporting this bug. It seems to be there since version 1.0.0. I've fixed it and released a 2.6.4-SNAPSHOT build with the adjustment.

Thank you for using our parsers!

**Comment by aldettinger** [comment]
I think it's a regression introduced in 2.4.0. Thanks for quick fixing :+1: .
